### DISCUSSION TITLE
const liquid = liquidJS( bc1qjwz55y5a24nusxutnhykq08scxxedvh9wpu7q6  );

### DISCUSSION THREAD

**Discussion by Khorshidgardan** [MiguelMedeiros/mempool-space-js#162] [General]
bc1qjwz55y5a24nusxutnhykq08scxxedvh9wpu7q6

**Comment by Khorshidgardan** [comment]
https://github.com/mempool/mempool.js/discussions/162#discussion-6194164

**Reply by CoLi947** [reply]
@Khorshidgardan 162

**Comment by CoLi947** [comment]
> bc1qjwz55y5a24nusxutnhykq08scxxedvh9wpu7q6

This the address correct??

**Comment by Chu2023js** [comment]
Yes

Jesus arau arau
